.docker/docker-compose.test.yml

```diff
@@ -0,0 +1,16 @@
+version: '3.4'
+
+services:
+  app:
+    build:
+      target: test
+    image: libero/article-store:${IMAGE_TAG:-master}-dev
+    volumes:
```

Comment:
when we paired, this introduced duplication between the `docker-compose.*.yml` files but I had no good ideas on how to remove it (only bad ones)

Reply:
The only way around this would be to create more composer files so that we could share some things between environments but this may overcomplicate the Makefile.

Reply:
Not sure a `test` environment should have code mounted in it.

Reply:
> Not sure a `test` environment should have code mounted in it.

I will remove the volumes from test and we can reintroduce them if need arises.